src/Definition/Redshift.php

```diff
@@ -58,9 +58,14 @@ public function getCompression()
     public function getSQLDefinition()
     {
         $definition =  $this->getType();
-        if ($this->getLength() && $this->getLength() != "") {
-            $definition .= "(" . $this->getLength() . ")";
+
+        $typesWithoutLength = ['TIMESTAMP', 'DATE', 'FLOAT', 'BOOLEAN'];
+        if (!in_array($this->getBasetype(), $typesWithoutLength)) {
```

Comment:
tak v tom případě by se tohle mělo řešit jinde a timestamp/date/float/boolean by tady neměl bejt validní s délkou. jestli ty datový typy někde extrahuješ, tak je odstraň tam, dává to smysl?